Fase-1/API/self-order-management/src/main/kotlin/com/fiap/selfordermanagement/core/domain/entities/Order.kt

```diff
@@ -1,11 +1,13 @@
 package com.fiap.selfordermanagement.core.domain.entities
 
 import com.fiap.selfordermanagement.core.domain.value_objects.Status
+import java.math.BigDecimal
 
 data class Order(
-    val total: Double,
-    val nickname: String,
+    val id: Long? = null,
```

Comment:
Esse ID como `long` pode ser um problema no futuro, não? Se não houver uma chave composta no BD para o dia, e ID for auto increment, poderíamos ter números de pedido enormes.

Sugestão: adicionar data do pedido.